app/src/pages/Play/index.tsx

```diff
@@ -198,7 +198,7 @@ function Play() {
       )}
       <Grid item>
         <TurnContextPanel
-          secondsLeft={Math.round(Math.max(secondsLeft, 0))}
```

Comment:
not the root fix... but

`Math.round(Math.max(NaN, 0))` --> `NaN`

this will now display 0 instead